hack/ginkgo-e2e.sh

```diff
@@ -81,7 +81,10 @@ if [[ -n "${CONFORMANCE_TEST_SKIP_REGEX:-}" ]]; then
   ginkgo_args+=("--skip=${CONFORMANCE_TEST_SKIP_REGEX}")
   ginkgo_args+=("--seed=1436380640")
 fi
-if [[ ${GINKGO_PARALLEL} =~ ^[yY]$ ]]; then
+if [[ -n "${GINKGO_PARALLEL_NODES:-}" ]]; then
+  ginkgo_args+=("--nodes=${GINKGO_PARALLEL_NODES}")
```

Comment:
yes - maybe just use a set number of nodes so that we get consistent results, regardless of where tests are run. (as you note, our tests are pretty CPU-light, so the number of cores shouldn't really matter.)